lib/screen/exhibitions/exhibitions_bloc.dart

```diff
@@ -43,24 +48,23 @@ class ExhibitionBloc extends AuBloc<ExhibitionsEvent, ExhibitionsState> {
       (event, emit) async {
         log.info('[ExhibitionBloc] getNextPageEvent:'
             'offset  ${state.currentPage * limit}');
-        List<Exhibition> proExhibitions =
+        List<Exhibition> exhibitions =
             await _feralFileService.getAllExhibitions(
```

Comment:
just a note: should be `getListExhibition`. `getAllExhibitions` make me confuse that it's called to get ALL exhibitions. :))